Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -149,18 +149,14 @@ extension NavigationMapView {
         // MARK: - NavigationComponentDelegate implementation
         
         func navigationViewDidLoad(_ view: UIView) {
-            navigationMapView.mapView.mapboxMap.onNext(.styleLoaded) { [weak self] _ in
+            navigationMapView.mapView.mapboxMap.onEvery(.styleLoaded) { [weak self] _ in
```

Comment:
@mapbox/navigation-ios do we know if it is safe to change to `onEvery`?

Reply:
As of #3332, the only thing left that was `onNext` will be the call to `localizeLabels()`, which does need to be reapplied every time the style changes.

Reply:
I'm also not sure that we should apply this change. Basically now any changes to the `NavigationMapView.mapView.showsTraffic`, which were made by the end user will be ignored.

Reply:
Any time the style changes, their change to `showsTraffic` would get blown away anyways.